app/scripts/controllers/network/networks.js

```diff
@@ -18,6 +20,18 @@ networks.networkList = {
     'exchanges': ['ShapeShift'],
     'buyUrl': '',
   },
+  [MORDEN]: {
+    'chainId': MORDEN_CODE,
+    'ticker': 'ETC',
+    'blockExplorerTx': 'https://gastracker.io/tx/[[txHash]]',
+    'blockExplorerAddr': 'https://gastracker.io/addr/[[address]]',
+    'blockExplorerToken': 'https://gastracker.io/token/[[tokenAddress]]/[[address]]',
+    'service': 'Gastracker.io',
+    'rpcUrl': 'https://web3.gastracker.io/morden',
```

Comment:
https://web3.gastracker.io/morden is not work correctly with MetaMask.
@pyskell // is there any official api-server/block explorer exist for the Morden testnet ?